app/api/items/get_children.go

```diff
@@ -283,7 +283,9 @@ func constructItemChildrenQuery(dataStore *database.DataStore, parentItemID, gro
 		dataStore, groupID, requiredViewPermissionOnItems, watchedGroupIDSet, watchedGroupID, columnList, columnListValues,
 		externalColumnList,
 		func(db *database.DB) *database.DB {
-			return db.Joins("JOIN items_items ON items_items.parent_item_id = ? AND items_items.child_item_id = items.id", parentItemID)
+			return db.Joins("JOIN items_items ON items_items.parent_item_id = ? AND items_items.child_item_id = items.id", parentItemID).
+				Joins("JOIN items AS parent_item ON parent_item.id = ?", parentItemID).
```

Comment:
If you introduce the `"JOIN items AS parent_item ON parent_item.id = ?", parentItemID`, IMO, you have to do the previous join `items_items` on `items_items.parent_item_id=parent_item.item_id`  instead of using the id again.

Reply:
fixed, it looks better